src/polynomial.cpp

```diff
@@ -0,0 +1,35 @@
+#include "polynomial.h"
+#include "add.h"
+
+namespace SymEngine {
+
+    Polynomial::Polynomial(const std::string& var, map_uint_mpz&& dict) :
```

Comment:
@certik In that case, it should not be  too difficult to switch to unordered map.